### ISSUE TITLE
NTR: [transcriptional interfering lncRNA activity]

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#22717]
Please provide as much information as you can: 

* **Suggested term label:**

This is a first stab at this term, open to suggestions:

transcriptional interfering lncRNA activity

* **Definition (free text)**

A molecular function that controls gene transcription by transcriptional interference with the synthesis of downstream messenger RNAs.

* **Reference, in format PMID:#######**

Genetic screen for suppression of transcriptional interference identifies a gain-of-function mutation in Pol2 termination factor Seb1.
PMID:34389684
see image:
https://www.pombase.org/reference/PMID:34389684

There are quite a few publications on this now, and a nice review:
Long RNA-Mediated Chromatin Regulation in Fission Yeast
and Mammals 
PMID:35055152 (published last week)

There are many more references in the review PMID: 35055152

* **Gene product name and ID to be annotated to this term**

nc-tgp1 (pombe)
nc-pho1  (pombe)
and others

* **Parent term(s)**

GO:0140110  transcription regulator activity
Is a

* **Any other information**

From the review PMID:35055152 
This is the relevant section:
**5. Regulation of Nearby Gene Transcription by Long RNAs**
"lncRNAs represent one approach to regulate neighboring genes. For instance, in S. cerevisiae, lncRNAs called
upstream-initiating transcripts (UPS) have recently been suggested to regulate transcription of nearby downstream rDNA genes [112]. In S. pombe, three example lncRNAs that have been studied in detail are nam1, rse1 and prt [60,63,113], which regulate the nearby genes byr2, ste11 and pho1, respectively. Studies on the functions of nam1 and rse1 lncRNAs were recently reviewed by Andric and Rougemaille [114] and will not be discussed."

Also reference in PMID:34389684
"lncRNA transcription traversing
the PHO mRNA promoters evicts the PHO gene-activating transcription factor Pho7 from its DNA binding sites (8)."

where ref (8) is this paper: Distinctive structural basis for DNA recognition by the fission yeast Zn2Cys6 transcription factor Pho7 and its role in phosphate homeostasis.
PMID:30212894

@hattrill 
@pgaudet 
@gantonazzo 
@RLovering 
@colinlog

**Comment by pgaudet** [comment]
Would this be the same activity as in https://github.com/geneontology/go-ontology/issues/23019?

**Comment by ValWood** [comment]
I don't think so. Which activity are you referring to? This one works by interference (competing transcription blocks the promoter of the neighbouring gene). I don't think there is any base-paring involved?

**Comment by colinlog** [comment]
Indeed, it is the disruption of the DNA double strand structure that is the proposed regulatory ' interference activity' in this case. By transcribing through the double strand DNA binding site for Pho7 the base paring of the Pho7 site is disrupted because, as it elongates an RNA (transcribes) the RNA polymerase always melts the double strand DNA template into a single strand bubble of about 17 nucleotides in length.  This is very similar to one of the MFs that was proposed for SNF2-type ATPases, namely DNA stripping (of Rad51 from DNA by SNF2-like enzyme Rad54 see PMID: [12453424] - the disruption of DNA-protein complexes by processive tracking along the DNA. However, here it is proposed for the RNA polymerase itself towards a dbTF. Perhaps the term could be a MF child of GO:0032986???  Note that there is some evidence emerging that also yeast vSWI/SNF can do this to dbTFs via tracking along the nucleosome free regions at promoters - and similarly GR dbTF DNA binding that was shown to be disrupted by human SWI/SNF in vitro see PMID:15099516. In this way, the remodelers are first recruited to dislodge nucl;eosomes and then they also end-up dislodging the factor that recruited them in the first place, maintaining a negative feed back loop to allow 'resetting' of the promoters.

**Comment by ValWood** [comment]
Did the RNA group discuss how to annotate these yet? I have quite a few and I'd like to annotate with some function  (mainly to  the upstream regulatory pathways operating through these ncRNAs to the target genes. since we know their mechanism of action (Interfering with the promoter of a protein coding gene)

I would use just "transcription regulator activity" since non of the children are suitable, but this is 

Restrictions
This term should not be used for direct annotation.

So currently I'm stuck.

**Comment by ValWood** [comment]
note to self also for nc-tgp1
PMID:25428589
Title | Long non-coding RNA-mediated transcriptional interference of a permease gene confers drug tolerance in fission yeast.

**Comment by pgaudet** [comment]
The term is GO:1903231 mRNA base-pairing translational repressor activity

**Comment by ValWood** [comment]
This is transcriptional interference? The ncRNA is inhibiting transcription, not translation?

**Comment by pgaudet** [comment]
the ncRNA is binding an mRNA that has already been transcribed, it's preventing the translation step

**Comment by ValWood** [comment]
OK fair point. I thought it was considered to be interference and I really wanted to connect the interfering RNA ncpho1 as a regulator of pho1 transcription  in the causal model:

<img width="687" alt="Screenshot 2022-12-13 at 18 11 54" src="https://user-images.githubusercontent.com/7359272/207412906-b435b15c-1da4-4177-b021-f3af3d50bb68.png">

but you are quite right, we are still missing the evidence that the nc-RNA is the agent and not the act if transcription itself. The alternative cleavage makes a shorter transcript that does  not interfere, but that doesn't rule out the fact that it could be the transcription itself. I am finding it hard to separate the 2 scenarios but I take your word from it.

I am not so worried about  mmi1(YTHDC1), it would be expected to have regulatory roles  since this just seems to be part of  the alternative cleavage and polyadenylation scenario which truncates nc-pho1 and increases pho1 expression. There are  a lot of players in the process.... will wait and see....

**Comment by ValWood** [comment]
I think there is a lot of evidence not that the lncRNA is the "agent"

i.e. 
<img width="516" alt="cryptic intron lncRNA" src="https://github.com/geneontology/go-ontology/assets/7359272/dd1999b8-b6fc-4e6f-8cec-c8f654bad198">

 I was going to just use "transcription regulator activity", but that is blocked for direct annotation

**Comment by hattrill** [comment]
@ValWood I was having a similar problem yesterday trying to describe the activity of a lncRNA that interacts with an NFkB transcription factor (PMID:37956726) - in some instances, promoting transcription (of it's own gene) and inhibiting the transcription of other NFkB targets. I put a note on it to revisit it.
Not sure of the meachanism. Authors suggest that it "sequesters" NFkB to it's own promoter, in the end I went for  the MF "protein sequestering activity" but I would have rather had one that was in-line with "transcription regulator activity".

Would be nice to link this up with the other ncRNA expression regulatory work and make it more specific. Perhaps we could put aside some time to come up with something more satisfactory.
